allure-commandline/src/main/java/io/qameta/allure/CommandLine.java

```diff
@@ -75,6 +75,10 @@ public static void main(final String[] args) throws InterruptedException {
 
     @SuppressWarnings("PMD.AvoidLiteralsInIfCondition")
     public Optional<ExitCode> parse(final String... args) {
+        if (args.length == 0) {
```

Comment:
Technically, yes, but then the test shouldParseEmptyArguments checks nothing. I'd vote to leave it as is, just to make this case explicit.